### PR TITLE
Feature/small tweaks and fixes

### PR DESCRIPTION
**Pull request by simaob**
Addresses various small changes listed on this PT Story: https://www.pivotaltracker.com/story/show/170605202

- tweaks on the home page hero section;
- adding links to the footer;
- adjusting the formatting of the litigations results page;
- adjusting the formatting of the legislation results page;
- small changes to the geography show page;
- small changes to litigation and legislation show page;

### CODE REVIEW

**Comment by aabdaab** [comment]
👀

**Comment by simaob** [comment]
Thanks for the feedback, Marta. I renamed it to `self_and_related_events`. I don't love it, but I think it's a little more clear.

File: app/models/legislation.rb [hunk 1]
```diff
@@ -113,4 +113,12 @@ def policy?
   def date_passed
     events.where(event_type: 'law_passed').first&.date
   end
+
+  def first_event
```
Comment (tsubik) [inline app/models/legislation.rb]: Maybe at some point worth to add concern like `HasEvents`

**Comment by tsubik** [comment]
I think that's a good name.

**Comment by aabdaab** [comment]
Thanks 👍it's comprehensive!

**Pull request merged**